app/src/processing/app/Base.java

```diff
@@ -98,6 +98,14 @@
     }
   }
 
+  static String nativeArch;
+  static {
+    nativeArch = System.getProperty("os.arch");
+    if (nativeArch == null) {
+      nativeArch = "";
```

Comment:
Yuck, no! `null` means something here, we should be testing for that rather than doing an empty string compare.